files/en-us/web/javascript/reference/global_objects/array/at/index.html

```diff
@@ -0,0 +1,121 @@
+---
+title: Array.prototype.at()
+slug: Web/JavaScript/Reference/Global_Objects/Array/at
+tags:
+  - Array
+  - JavaScript
+  - Method
+  - Prototype
+  - Reference
+  - polyfill
+  - at
+  - Experimental
+---
+<div>{{JSRef}}</div>
+
+<p class="summary">The <strong><code>at()</code></strong> method takes an integer value and returns the item at that index, allowing for negative integers which count back from the last item.</p>
+
+<div>{{EmbedInteractiveExample("pages/js/array-at.html","shorter")}}</div>
```

Comment:
Renders without scrollbar for me if you remove shorter.

```suggestion
<div>{{EmbedInteractiveExample("pages/js/array-at.html")}}</div>
```